corehq/apps/reports/static/reports/js/bootstrap5/datatables_config.js

```diff
@@ -300,12 +300,11 @@ function convertDate(k) {
     return new Date(m[1]);
 }
 
+// todo on reports where this is used, see if this is still relevant
+// see https://datatables.net/new/1.10#Improved-built-in-sorting-and-type-detection
```

Comment:
We can likely remove these sorts for now and ask QA to check sorting when a report is migrated, and re-implement if/when needed.

Reply:
updated here https://github.com/dimagi/commcare-hq/pull/35648/commits/836c7e6be1d38885a03cf462762f06a9d77166c5